observing/schedule.py

```diff
@@ -60,6 +61,32 @@ def get_sched():
     return dd, dd2
 
 
+def check_sched(sched):
+    """ Check if sched is requesting a beam that is already scheduled or submitted
+    """
+
+    dd, dd2 = get_sched()
```

Comment:
Can you make these the arguments to the function?

Reply:
I think it is more clear to have `is_conflicted` and have it take a single argument that is tested for conflict.